src/components/WeekWeatherData/index.js

```diff
@@ -1,40 +1,83 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import styled from 'styled-components';
+import { format } from 'date-fns';
+import { WeekCard } from './WeekCard';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCityLatLongImageUrban } from '../../helper/api';
+import { fetchWeather, setImage } from '../../store/slices/weatherSlice';
 import { Dropdown } from '../UI/Dropdown';
+import { enUS, srLatn } from 'date-fns/locale';
+import { useTranslation } from 'react-i18next';
 
-const cities = [
-  {
-    id: '1',
-    name: 'NORWAY',
-  },
-  {
-    id: '2',
-    name: 'BELGRADE',
-  },
-
-  {
-    id: '3',
-    name: 'FRANCE',
-  },
-
-  {
-    id: '4',
-    name: 'SAD',
-  },
-];
+const langMap = {
+  en: enUS,
+  sr: srLatn,
+};
 
 export function WeekWeatherData() {
+  const { i18n } = useTranslation();
+  const dispatch = useDispatch();
+
+  const { cities, fetching, image, weekForecast } = useSelector(
+    (state) => state.weather
+  );
+
+  const date = format(new Date(), 'dd MMMM', {
+    locale: langMap[i18n.language],
+  });
+
+  const handleCitySelect = async (e) => {
+    const urbanApi = e?.href || e;
+    const { lat, lon, image } = await getCityLatLongImageUrban(urbanApi);
+    dispatch(setImage(image));
+    dispatch(fetchWeather({ lat, lon }));
+  };
+
+  useEffect(() => {
+    if (cities.length > 0) {
+      handleCitySelect(cities[0].href);
```

Comment:
If we pass here `cities[0]`, can we then pick out the `href` in the `handleCitySelect` function?